docs/topics/secured-mission-intro.adoc

```diff
@@ -31,28 +35,28 @@ a|
 The configured **clients accept any callback url**.
 
 a|
-To avoid having a custom configuration per runtime we avoid the verification of the callback as mandated by the OAuth2 specification.
+This example does not perform verification of the callback as mandated by the OAuth2 specification. A default configuration suitable for all runtimes was used.
 
 a|
-An application-specific callback URL should be provided with a valid domain name.
+In production, an application-specific (custom) callback URL with a valid domain name should be provided.
```

Comment:
Should be provided by whom? I believe the passive voice here hinders comprehension. Can you please add the actor in this sentence? Thanks.

Reply:
Didnt intend to make it passive voice, just add prouction... will restore origninal

Reply:
I don't think @tradej is asking you to restore the original either, @lavernw1. "Should be provided" is present in both. It's unclear if you're asking the reader to provide the URL themselves, or letting them know that a URL should automatically be provided by something else. If you're asking the user to perform an action, tell them explicitly with active voice/an imperative verb.